src/NServiceBus.Testing/TestingContext.cs

```diff
@@ -0,0 +1,82 @@
+﻿namespace NServiceBus.Testing
+{
+    using System;
+    using System.Collections.Concurrent;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using NServiceBus.Testing.ExpectedInvocations;
+
+    class TestingContext : TestableMessageHandlerContext
```

Comment:
I'm on the fence about this move. What would be the harm of exposing `TimeoutMessage` also on the `TestableMessageHandlerContext`? I can't see any harm. It would even make it more usable when I don't use the testing framework fluent syntax provided for the sagas. 

If we come to the conclusion that we can expose `TimeoutMessages` then this class would only contain the expected invocations and the validate method which then probably makes this class questionable.